src/cyclic_shifft/cyclic_shift.c

```diff
@@ -0,0 +1,15 @@
+#include "cyclic_shift.h"
+unsigned int cyclicShift(unsigned int num, unsigned int n)
```

Comment:
По стайлгайду надо так:
```suggestion
unsigned int cyclicShift(unsigned num, unsigned n)
```